src/main/java/uk/gov/ida/verifyserviceprovider/factories/VerifyServiceProviderFactory.java

```diff
@@ -101,8 +97,11 @@ public GenerateAuthnRequestResource getGenerateAuthnRequestResource() throws Exc
     public TranslateSamlResponseResource getTranslateSamlResponseResource() {
         if(configuration.getMsaMetadata().isPresent()) {
             return getTranslateMatchingSamlResponseResource();
-        } else{
-            return getTranslateNonMatchingSamlResponseResource();
+        }
+        if (isEidasEnabled()){
```

Comment:
isn't this going to make VSP to be eidas-only or idp-only?

Reply:
Yes it wood. Good spot, no changes to this file should have been in here.